index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <div class="header__bg">
+        <img src="img/bg.png" alt="bg" class="header__bg-content" />
+      </div>
+      <div class="header__flex-container">
+        <div class="header__flex-container-logobox">My Logo</div>
```

Comment:
- [ ] Logo is supposed to be a link because users generally click on logo to go to home page. Let's wrap logo with `<a>` tag